_posts/en/posts/2019-01-15-notice-cve-2018-20587.md

```diff
@@ -0,0 +1,69 @@
+---
+title: CVE-2018-20587 Advisory and Full Disclosure
+name: cve-2018-20587-full-disclosure
+id: en-cve-2018-20587-full-disclosure
+lang: en
+type: posts
+layout: post
+share: true
+
+## If this is a new post, reset this counter to 1.
+version: 1
+
+excerpt: >
+  Disclosure of the impact and mitigation of CVE-2018-20587, affecting all
+  released versions of Bitcoin Core, including the latest 0.17.1.
+---
+
+Full disclosure
+===============
+
+CVE-2018-20587 is a Incorrect Access Control vulnerability that affects all currently released versions of Bitcoin Core, including the latest 0.17.1.
+
+You may be affected by this issue if you have the RPC service enabled (default, with the headless bitcoind), and other users have access to run their own services on the same computer (either local or remote access).
+If you are the only user of the computer running your node, you are not affected.
```

Comment:
This is understated - sure, the discovery of this particular bug doesn't change this a ton, but you still have a lot of risk from other software running on your computer which can access your RPC port. Can we make this a much broader "risks when using RPC port" post? There was also the previous discussion a few months ago where some wallet was telling users to expose their RPC port to the internet which merits discussion.